README.md

```diff
@@ -1,3 +1,5 @@
+### This plugin is forked from cordova device plugin. This plugin serves the following issue
```

Comment:
With this change it is impossible for the original plugin to review and merge your PR. Could you maybe create a branch on the commit _before_ this one that added this text and change the PR to use that branch? Then we can still merge it.